galaxy.yml

```diff
@@ -25,7 +25,6 @@ tags:
   - ims
   - zos_ims
   - z_and_cloud_modernization_stack_operator
-  - zoscb
```

Comment:
We need to keep this tag in our version of `galaxy.yml` but we should add a line in the customization guide explaining that this tag should be removed otherwise the customized collection can show up in z/OS Cloud Broker's Galaxy Import page (if the customized collection is uploaded to Galaxy)

Reply:
i updated galaxy.yml with variable descriptions and added a note about the zoscb tag

Reply:
@jashnani let me know if you have seen my last comment